CHANGELOG.rst

```diff
@@ -1,5 +1,13 @@
 This file includes migration steps for each release of CGXP.
 
+Version 1.6
+===========
+
+1. We start to use the MapFishPrint version 3 then in the ``cgxp_print`` 
```

Comment:
GeoMapFish 1.6 uses MapFishPrint version 3. In the configuration of the ``cgxp_print`` plugin, ``fields`` is deprecated and should be replaced by ``additionalAttributes``.